pennylane_lightning/core/src/simulators/lightning_qubit/measurements/MeasurementsLQubit.hpp

```diff
@@ -581,6 +633,8 @@ class Measurements final
     }
 
   private:
+    const std::unordered_map<std::string, ExpValFunc> expval_funcs_;
```

Comment:
Why not have this variable definition together with the others? Or even better, what do you think about merging the two private sections?

Reply:
I moved the variable definition to the first private section. Regarding having two private sections, my intention was to minimize the modifications to the existing code. I found other examples with more than one `public` or `privat` sections in the codebase, such as [StateVectorLQubitManaged.hpp](https://github.com/PennyLaneAI/pennylane-lightning/blob/master/pennylane_lightning/core/src/simulators/lightning_qubit/StateVectorLQubitManaged.hpp), so I thought this might be part of coding style.